src/lib/ledger_catchup/README.md

```diff
@@ -0,0 +1,39 @@
+# Notes on super catchup run logic
+
+### Quick definitions
+
+- Block recap: a block contains a hash of the current ledger, and a `staged_ledger_diff`, which is effectively a list of transactions that change the previous staged ledger to the current block's staged ledger. A transaction is of the form "acct a sends X mina tokens to acct B". It doesn't tell you the balances of acct A and B. The block does not contain the ledger in it's entirety, nor can any account balances be looked up.  https://docs.minaprotocol.com/en/architecture/whats-in-a-block#staged-ledger-diff
+
+- Breadcrumb: a breadcrumb is a block which has been fully validated and given an attached and connected ledger mask which connects to the previous mask which eventually connects to a ledger, and so an attached mask in essence is a full ledger. breadcrumbs are the "nodes" in a transition frontier. breadcrumbs can only exist if the parent breadcrumb already exists in the transition frontier, because the parent is necessary for validating and of course a fully attached ledger mask can only exist if there exists a previous ledger.  the transition frontier, being made of breadcrumbs, is therefore always fully connected.
+
+- Transition frontier: the transition frontier is a tree of breadcrumbs with max height of `k`, where `k` is a constant defined by the crypto team, currently set at `k = 209`.  the transition frontier is the very most recent `k` number of blocks, as well as all the forks, or sidechains, off of the current longest chain.  the block-chain path through the transition frontier with the best chain quality metric is considered the current canonical chain, and the tip of this canonical chain is called the best-tip.  
+
+- Bootstrap, high level: bootstrap is the process of using zk-snarks to validate the validity of the chain, from the last known block (which would be the genesis block for fresh daemons just starting up) to the best tip.
```

Comment:
A better description would be "bootstrap is the process by which nodes join the network, synchronizing to the network's best tip".